execution/executor-benchmark/src/pipeline.rs

```diff
@@ -17,6 +17,13 @@ use std::{
     thread::JoinHandle,
 };
 
+#[derive(Clone, Debug)]
+pub struct PipelineConfig {
```

Comment:
what's wrong with using the Opt directly? (or derive Parser on this)

Reply:
nothing, just thought it is cleaner to not have command line related things in this file, or have this one depend on lib